src/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -337,15 +337,18 @@ public void addNotification(int accountId, int chatId, int msgId) {
                 return;
             }
 
+            NotificationManagerCompat notificationManager = NotificationManagerCompat.from(context);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && !notificationManager.areNotificationsEnabled()) {
+                return;
```

Comment:
shouldn't this go below the next `if` ?? otherwise in-chat sounds will not play :thinking: